code/stats/scoring.cpp

```diff
@@ -105,12 +105,10 @@ void parse_rank_tbl()
 		required_string("#End");
 
 		// be sure that all rank points are in order
-#ifndef NDEBUG
```

Comment:
Warnings still have an impact in release builds (if too many occur, a pop-up tells the player how many there were and that the mod may be broken); a Warning should never be hidden behind a debug build check.